dpuVendor.py

```diff
@@ -39,15 +39,12 @@ def build_push_start(self, acc: host.Host, imgReg: ImageRegistry) -> None:
         # If p4 pod already exists from previous run, kill this first.
         acc.run(f"podman ps --filter ancestor={local_img} --format '{{{{.ID}}}}' | xargs -r podman kill")
 
-        # Temporarily use a container until issue with p4 running as a pod is resolved: https://issues.redhat.com/browse/IIC-465
-        # start_p4_pod(acc, client, local_img)
-        self.start_p4_container(acc, local_img)
+        start_p4_pod(acc, client, local_img)
 
     def start_p4_pod(self, acc: host.Host, client: K8sClient, image: str) -> None:
         self.configure_p4_hugepages(acc)
 
         logger.info("Manually starting P4 pod")
-        acc.run_or_die("mkdir -p /opt/p4/p4-cp-nws/var/run/openvswitch")  # WA https://issues.redhat.com/browse/IIC-421
```

Comment:
This issue should have been fixed by https://github.com/intel/ipu-opi-plugins/pull/278 hence removing